anomaly-detection/src/main/java/io/pravega/anomalydetection/event/producer/ControlledSourceContextProducer.java

```diff
@@ -50,15 +53,16 @@ public void run(SourceContext<Event> ctx) throws Exception {
 				if(event.isPresent()) {
 					ctx.collect(event.get());
 					invalidCount += 1;
-					LOG.info("*** Emitting invalid event: [{}], total count so far: [{}] ", event, invalidCount);
+					System.out.println(String.format(
```

Comment:
The `ControlledSourceContextProducer` is interactive and works only with the local Flink environment.  As I was using it, I felt that the interactive outputs should be sent directly to the console, regardless of the log verbosity settings.